### PR TITLE
Fix cmake python build

### PR DESCRIPTION
**Pull request by chuckatkins**
Depends on #12 being merged first

### CODE REVIEW

File: examples/hello/bpTimeWriter/helloBPTimeWriter.py [hunk 1]
```diff
@@ -22,18 +22,18 @@
...
-adios = ADIOS(comm, adiosDebugON)
+adios = adios2.ADIOS(comm, adios2.adiosDebugON)
 
 # IO
 bpIO = adios.DeclareIO("BPN2N")
 
 # Variables
 bpArray = bpIO.DefineVariable("bpArray", [size * nx], [rank * nx], [nx],
-                              adiosConstantDims)
+                              adios2.adiosConstantDims)
```
Comment (williamfgc) [inline examples/hello/bpTimeWriter/helloBPTimeWriter.py:32]: I see you're using the module explicitly, I will change adiosConstantDims to ConstantDims in the glue code

File: examples/hello/bpTimeWriter/helloBPTimeWriter.py [hunk 2]
```diff
@@ -22,18 +22,18 @@
...
 bpTimeStep = bpIO.DefineVariable("bpTimeStep")
 
 # Engine
-bpFileWriter = bpIO.Open("myArray.bp", adiosOpenModeWrite)
+bpFileWriter = bpIO.Open("myArray.bp", adios2.adiosOpenModeWrite)
```
Comment (williamfgc) [inline examples/hello/bpTimeWriter/helloBPTimeWriter.py:36]: same here -> adiosOpenModeWrite to OpenModeWrite in glue

File: examples/hello/bpWriter/helloBPWriter.py [hunk 3]
```diff
@@ -8,24 +8,24 @@
...
-from adios2py import *
+import adios2
 import numpy as np
 
 # User data
 myArray = np.array([1., 2., 3., 4., 5., 6.])
 
 # ADIOS
-adios = ADIOS(MPI.COMM_WORLD, adiosDebugON)
+adios = adios2.ADIOS(MPI.COMM_WORLD, adios2.adiosDebugON)
```
Comment (williamfgc) [inline examples/hello/bpWriter/helloBPWriter.py:18]: adiosDebugOn to DebugOn

**Review by williamfgc — APPROVED** [review]

**Comment by chuckatkins** [comment]
If I understand my python conventions correctly (which, I think I do, but maybe I don't), explicitly using the modules is a more pythonic way to use it.  Things like `from adios2 import *` are typically frowned upon in the Python world.

**Pull request merged**